api/src/main/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimController.java

```diff
@@ -69,6 +72,19 @@ public ClaimDTO retrieveClaimById(@PathVariable("id") UUID id) {
         return claimToClaimDTOConverter.convert(claim);
     }
 
+    @PostMapping("/search")
+    @ApiOperation("Retrieve Claims.")
+    @ApiResponses({@ApiResponse(code = 400, message = "Bad request", response = ErrorResponse.class)})
+    public ResponseEntity<List<ClaimResponseDTO>> retrieveAllClaims(@RequestBody @Valid @ApiParam("retrieve claims") Map<String, String> json) {
+        log.debug("Retrieve claims");
+
+        Iterable<Claim> claims = claimRepository.findAll();
```

Comment:
ideally we shouldn't call repository directly from controller. if possible please move this to ClaimService. And also i would prefer to return that as List<Claim> instead of Iterable.  @adeel718 what do you think?